scriptmodules/ports/darkplaces-quake.sh

```diff
@@ -13,11 +13,12 @@ rp_module_id="darkplaces-quake"
 rp_module_desc="Quake 1 engine - Darkplaces Quake port with GLES rendering"
 rp_module_licence="GPL2 https://raw.githubusercontent.com/xonotic/darkplaces/master/COPYING"
 rp_module_section="opt"
-rp_module_flags="!mali !kms"
+rp_module_flags="!mali"
 
 function depends_darkplaces-quake() {
     local depends=(libsdl2-dev libjpeg-dev)
     isPlatform "rpi" && depends+=(libraspberrypi-dev)
```

Comment:
Should this be switched to videocore  or?

Reply:
Yep, somehow missed that one. Patch has been updated, thanks!